packages/http-client-csharp/generator/SamplePlugin/src/SampleCodeModelPlugin.cs

```diff
@@ -14,6 +14,9 @@ public class SampleCodeModelPlugin(GeneratorContext context) : ClientModelPlugin
     {
         public override SamplePluginTypeFactory TypeFactory { get; } = new SamplePluginTypeFactory();
 
-        public override OutputLibrary OutputLibrary { get; } = new SamplePluginOutputLibrary();
+        public override void Configure()
+        {
+            OutputLibrary.AddVisitor(new SamplePluginOutputLibraryVisitor());
```

Comment:
I almost feel like Configure should take in the visitors and do this internally?

Reply:
This is one hook that lets customers add the visitor. They can also derive from OutputLibrary and override GetVisitors.